### DISCUSSION TITLE
Welcome to DL-Simplified Discussions!

### DISCUSSION THREAD

**Discussion by abhisheks008** [abhisheks008/DL-Simplified#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aman-tech45** [comment]
Hey Aman here, I am an engineering student, and I built ML models

**Comment by adityamajji1654** [comment]
Hi @abhisheks008 

I'm looking at the issues and wondering if I can work on the ones marked "status: assigned" or if I should only pick those labeled "status: up for grabs."

**Comment by Amarta113** [comment]
@abhisheks008 can I add FCN model trained on fashion_mnist dataset

**Comment by DebanKsahu** [comment]
@abhisheks008 I am having some problems during cloning your forked repo in my local server. 
remote: Enumerating objects: 18880, done.
remote: Counting objects: 100% (592/592), done.
remote: Compressing objects: 100% (437/437), done.
error: RPC failed; curl 56 Recv failure: Connection reset by peer
error: 8116 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
could you help

**Reply by Amarta113** [reply]
use github codespaces
